uwebsockets/protocol.py

```diff
@@ -58,7 +66,7 @@ class Websocket:
     is_client = False
 
     def __init__(self, sock):
-        self._sock = sock
+        self.sock = sock
```

Comment:
Why change this?

Reply:
The reason I did this is that I was trying to port a project that uses [websocket-client](https://github.com/websocket-client/websocket-client) and in this case the client has `sock` as attribute. Using the same attribute name makes it more compatible. 
If that's the prefered way, I can revert to `_sock` and modify my port of the other library instead. Please let me know which is best.

Reply:
Okay that's a good reason.